components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -0,0 +1,264 @@
+#include "atmosphere_nudging.hpp"
+
+namespace scream
+{
+
+  //using namespace spa;
+// =========================================================================================
+NUDGING::NUDGING (const ekat::Comm& comm, const ekat::ParameterList& params)
```

Comment:
I would not use all uppercase names. Uppercase are associated with either acronyms or compile-time constants. I would just use `Nudging`.

Reply:
Fixed